src/constants.ts

```diff
@@ -57,3 +59,4 @@ export const MIGRATIONS_DIR = path.join(
 export const CONTENTFUL_ALIAS = INPUT_CONTENTFUL_ALIAS || DEFAULT_CONTENTFUL_ALIAS;
 export const DELAY = Number(INPUT_DELAY || DEFAULT_DELAY);
 export const MAX_NUMBER_OF_TRIES = Number(INPUT_MAX_NUMBER_OF_TRIES || DEFAULT_MAX_NUMBER_OF_TRIES);
+export const WITH_DELIVERY_TOKEN = booleanOr(INPUT_CREATE_CDA_TOKEN, DEFAULT_CREATE_CDA_TOKEN)
```

Comment:
Have generally been going with the pattern that the exported const matches the input name, so in this case, `CREATE_CDA_TOKEN`